### PR TITLE
new Pricing page

### PR DESCRIPTION
**Pull request by hasparus**
Added a `/pricing` page with the new query cost estimation slider.

Please let me know if the pricing list items make sense (or feel free to edit them on this branch).

### CODE REVIEW

File: packages/web/docs/src/components/pricing.tsx [hunk 1]
```diff
@@ -158,38 +157,92 @@ export function Pricing(): ReactElement {
...
+                <>
+                  <PlanFeaturesListItem>
+                    <Tooltip content={USAGE_DATA_RETENTION_EXPLAINER}>
+                      <strong>Custom</strong> data retention
+                    </Tooltip>
+                  </PlanFeaturesListItem>
+                  <li className="mb-2 mt-8">Everything in Pro, plus:</li>
+                  <PlanFeaturesListItem>Dedicated Slack channel for support</PlanFeaturesListItem>
+                  <PlanFeaturesListItem>White-glove onboarding</PlanFeaturesListItem>
+                  <PlanFeaturesListItem>Improved pricing as you scale</PlanFeaturesListItem>
```
Comment (saihaj) [inline packages/web/docs/src/components/pricing.tsx:174]:
```suggestion
                  <PlanFeaturesListItem>Bulk volume discount</PlanFeaturesListItem>
```

File: packages/web/docs/src/components/pricing-page.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+function PricingPageHero({ className }: { className?: string }) {
+  return (
+    <div
+      className={cn(
+        'bg-beige-100 relative isolate flex max-w-[90rem] flex-col gap-6 overflow-hidden rounded-3xl px-4 py-6 sm:py-12 md:gap-8 lg:py-24',
+        className,
+      )}
+    >
+      <DecorationIsolation>
+        <ArchDecoration className="pointer-events-none absolute left-[-46px] top-[-20px] size-[200px] rotate-180 md:left-[-60px] md:top-[-188px] md:size-auto" />
```
Comment (dimaMachina) [inline packages/web/docs/src/components/pricing-page.tsx:42]:
```suggestion
        <ArchDecoration className="pointer-events-none absolute left-[-46px] -top-5 size-[200px] rotate-180 md:left-[-60px] md:top-[-188px] md:size-auto" />
```

File: packages/web/docs/src/components/pricing.tsx [hunk 3]
```diff
@@ -61,88 +59,89 @@ function Plan(props: {
...
+        {children}
 
         <div
           // the padding is here so `overflow-auto` doesn't cut button hover states
           className="-mx-2 overflow-auto px-2"
         >
-          <div className="mt-16 flex min-w-[1000px] flex-row items-stretch gap-8 px-6 lg:mt-24 lg:gap-10 xl:gap-12 xl:px-0">
+          <div
+            className={cn(
+              'flex min-w-[1000px] flex-row items-stretch gap-8 px-6 lg:gap-10 xl:gap-12 xl:px-0',
```
Comment (dimaMachina) [inline packages/web/docs/src/components/pricing.tsx:86]:
```suggestion
              'flex min-w-[1000px] items-stretch gap-8 px-6 lg:gap-10 xl:gap-12 xl:px-0',
```

File: packages/web/docs/src/components/slider.tsx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+ */
+function polyfillSlider(element: HTMLElement, cssProperty: `--${string}`) {
+  if (CSS.supports('(animation-timeline: view()) and (animation-range: 0 100%)')) {
+    return;
+  }
+
+  const input = element.querySelector('[type=range]') as HTMLInputElement;
+
+  const sync = () => {
+    const val = (Number(input.value) - Number(input.min)) / (Number(input.max) - Number(input.min));
```
Comment (dimaMachina) [inline packages/web/docs/src/components/slider.tsx:105]: is this same input element?
```suggestion
  const sync = (event) => {
    const input = event.target
    const val = (Number(input.value) - Number(input.min)) / (Number(input.max) - Number(input.min));
```

**Pull request merged**